config/locales/views/en.yml

```diff
@@ -700,5 +712,25 @@ en:
         unrestricted: Show to all
     show:
       unposted_deletion_notice_html: This work is a draft and has not been posted. The draft will be <strong>scheduled for deletion</strong> on %{deletion_date}.
+    work_approved_children:
+      inspired:
+        restricted_html: "[Restricted Work] by %{creator_link}. Log in to view."
+        revealed_html: "%{work_link} by %{creator_link}."
+        title: Works inspired by this one
+      translated_to:
+        restricted_html: "[Restricted Work] by %{creator_link} (translation into %{language}) Log in to view."
+        revealed_html: "%{work_link} by %{creator_link} (translation into %{language})"
+    work_header_notes:
+      inspired_by:
+        restricted_html: Inspired by [Restricted Work] by %{creator_link}. Log in to view.
+        revealed_html: Inspired by %{work_link} by %{creator_link}.
+        unrevealed: Inspired by a work in an unrevealed collection.
+      translated_to:
+        restricted_html: 'Translation into %{language} available: [Restricted Work] by %{creator_link}. Log in to view.'
```

Comment:
Calling `i18n-tasks normalize` [converts ](https://github.com/glebm/i18n-tasks/issues/248) this to single quotes.